chatbot.py

```diff
@@ -17,13 +18,15 @@ def setup():
 
 def message_to_bot(H,clf,learn_response):
     if learn_response == 2:
-        add_to_maps_database(H,"")
+        #add_to_maps_database(H,"")
+        location_Dict["origin"]=H
         B = "Can you help me with the destination location?"
         learn_response = 3
         return B,learn_response
     if learn_response == 3:
-        add_to_maps_database("",H)
-        origin,destination = get_from_maps_database()
+        location_Dict["destination"]=H
+        origin = location_Dict["origin"]
+        destination = location_Dict["destination"]
```

Comment:
Refactor the statements into a single statement `origin, destination = location_dict["origin"], location_dict["destination"]`